ltm.go

```diff
@@ -1951,7 +1975,9 @@ func (b *BigIP) CreateMonitor(name, parent string, interval, timeout int, send,
 		SendString:    send,
 		ReceiveString: receive,
 	}
-
+	if len(partition) > 0 {
+		config.Partition = partition
```

Comment:
same here, no possibility to without partition specified in POST.

Reply:
Get and Delete Pool   100% support  "/partition/nodename"  we have implemented this with 100% success

Again you need to look at  AddMonitor   not Create if you want to pass full spec  including partition